index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+  <script src="script.js"></script>
+  <title>Library</title>
+</head>
+<body class="container">
+  <h1>Add Book To Library</h1>
+  <form>
+    <div class="mb-3">
+      <input type="text" id="newBookName" placeholder="Book name">
+    </div>
+    <div class="mb-3">
+      <input type="text"  id="newBookAuthor" placeholder="Author">
+    </div>
+    <div class="mb-3">
+      <input type="number" id="pages" >
```

Comment:
Please add a placeholder also to this input so the users know what should they input here, it might be a little confusing otherwise on what should they write in that input. [Pages Input](https://prnt.sc/15ivd2g)